nntrainer/graph/graph_core.h

```diff
@@ -205,6 +205,26 @@ class GraphCore {
    */
   void addLossToSorted();
 
+  void addInputNode(std::shared_ptr<GraphNode> &node) {
+    input_list.push_back(node);
+  }
+
+  const std::shared_ptr<GraphNode> &getInputNode(unsigned int idx) const {
```

Comment:
Please add doxygen for the functions added.

Reply:
I wonder how doxygen CI passed for this PR.

Reply:
Added doxygen